extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -177,14 +210,24 @@ const renderOutlinkPageAttachmentUI = (
       'i-amphtml-story-outlink-page-attachment-outlink-chip-no-image'
     );
   } else {
-    const ctaImgEl = win.document.createElement('div');
-    ctaImgEl.classList.add('i-amphtml-story-outlink-page-attachment-img');
     if (openImgAttr) {
+      const ctaImgEl = win.document.createElement('div');
+      ctaImgEl.classList.add('i-amphtml-story-outlink-page-attachment-img');
       setImportantStyles(ctaImgEl, {
         'background-image': 'url(' + openImgAttr + ')',
       });
+      chipEl.prepend(ctaImgEl);
+    } else {
+      const linkImage = htmlFor(
+        attachmentEl
+      )`<svg class="i-amphtml-story-page-open-attachment-link-icon" xmlns="http://www.w3.org/2000/svg" width="24" height="24">
```

Comment:
Can the SVG be living in the CSS instead, and added through a class?

Reply:
The CSS needs to change the color based on the accent color (the same color as the text), and CSS `background-image` can't access CSS vars or be changed through JS. The simplest workaround is to have the SVG in the HTML allowing the use of the CSS vars, unless you had another way fo doing this (like tinting background-images or other sorcery)